google/cloud/ndb/model.py

```diff
@@ -2414,11 +2442,48 @@ def _from_base_type(self, value):
             decompressed.
         """
         if self._compressed and not isinstance(value, _CompressedValue):
+            if not value.startswith(b"x\x9c"):
```

Comment:
A comment with a reference URL to document for this file signature would be good. Maybe declare it as a constant and have the documentation at the constant declaration, so we only need to document it once.